### PR TITLE
fix(onboarding): A few fixes for the new onboarding flow

### PR DESCRIPTION
**Pull request by PixelDust22**
- The client-state scope should be org instead of member. Otherwise we may create projects for the same platform twice.
- Fix a problem with emotion styled component passing unwanted props down, causing problems.
- Fixed a variable naming problem

### CODE REVIEW

**Review by AniketDas-Tekky — COMMENTED** [review]
`owners-api` shouldn't be needed for this change. No need to wait on approval from us.

**Review by scefali — APPROVED** [review]

**Comment by Neo-Zhixing** [comment]
@AniketDas-Tekky This was requested automatically due to CODEOWNER rules. Sorry for the noise. I'll update the rules in this PR.

**Pull request merged**
